apps/src/applab/PlaySpaceHeader.jsx

```diff
@@ -65,12 +65,12 @@ var PlaySpaceHeader = React.createClass({
         </table>
       </div>
     );
-  },
+  }
 
-  shouldHideToggle: function () {
+  shouldHideToggle() {
     return this.props.isShareView || !this.props.hasDesignMode;
-  },
-});
+  }
```

Comment:
needs to be bound?

Reply:
As far as I could tell, this helper is always invoked directly from `this` (i.e. `this.shouldHideToggle()`) and never passed around as a callback (e.g. `onClick={this.shouldHideToggle}`) so there isn't a need to bind it.